### PR TITLE
Remove the Travis tests on Python 2.6

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by asaunier** [comment]
Looks good.
If Python 2.6 is no longer supported, it should be removed from the requirements at http://docs.camptocamp.net/c2cgeoportal/master/integrator/requirements.html as well.

**Comment by sbrunner** [comment]
Effectively, updated :-)

File: doc/integrator/requirements.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ To install a GeoMapFish application you need to have the following
 components installed on your system:
 
 * **Git** (preferably to other revision control systems)
-* **Python** 2.6, 2.7 (2.5 or 3.x are not supported) with development files (``python-dev``)
+* **Python** 2.7 (2.5 or 3.x don't works, 2.6 should work but is not supported) with development files (``python-dev``)
```
Comment (asaunier) [inline doc/integrator/requirements.rst]: don't works => don't work

**Comment by asaunier** [comment]
Still a small typo left, else OK for merging.

**Pull request merged**